packages/@react-native-windows/cli/src/index.ts

```diff
@@ -18,6 +18,11 @@ import {runWindowsCommand} from './runWindows/runWindows';
 import {dependencyConfigWindows} from './config/dependencyConfig';
 import {projectConfigWindows} from './config/projectConfig';
 
+// This is effectively a type-only import because Options is an interface. This
+// is not a type import because prettier < 2.0 creating parse errors for type
+// imports. Non-type imports should not be added to this.
+import {Options} from 'yargs';
```

Comment:
Re-import as YargsOptions

Reply:
Also need to double check that this does not in fact try to import yargs at runtime.

Reply:
Either that, or we can consolidate further to yargs imported, and a source of truth within the package.